src/commands/xread.json

```diff
@@ -67,6 +67,33 @@
                     }
                 ]
             }
-        ]
+        ],
+        "reply_schema": {
+            "anyOf": [
+                {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "array",
+                        "items": {
+                            "type": "array",
+                            "minItems": 2,
+                            "maxItems": 2,
+                            "items": [
+                                {
+                                    "type": "string"
+                                },
+                                {
+                                    "type": "array"
+                                }
+                            ]
+                        }
+                    }
+                },
+                {
+                    "description": "when a timeout occurs",
```

Comment:
in case there are several possible schemas, they depend on the given args. please mention it in the description.